pymcspearman/pymckendall.py

```diff
@@ -0,0 +1,128 @@
+"""
+pymckendall.py
+
+Python implementation of Isobe, Feigelson & Nelson (1986) method for calculating Kendall
+tau rank correlation coefficient with uncertainties on censored data (upper/lowerlimit).
+
+"""
+
+import numpy as np
+from matplotlib import pyplot as plt
+import pandas as pd
```

Comment:
I don't think pandas is used, so you can remove this import?

Reply:
Same with the matplotlib import

Reply:
removed unused packages